packages/component/src/Middleware/ActivityStatus/Timestamp.js

```diff
@@ -16,17 +16,15 @@ const Timestamp = ({ activity: { timestamp }, 'aria-hidden': ariaHidden, classNa
       'Web Chat: styleSet.timestamp is being deprecated. Please use styleSet.sendStatus. This deprecation migration will be removed on or after December 31, 2021.'
     );
 
-  if (!timestamp) {
-    return false;
-  }
-
   return (
-    <span
-      aria-hidden={ariaHidden}
-      className={classNames((timestampStyleSet || sendStatusStyleSet) + '', (className || '') + '')}
-    >
-      {timestampFormat === 'relative' ? <RelativeTime value={timestamp} /> : <AbsoluteTime value={timestamp} />}
-    </span>
+    !!timestamp && (
```

Comment:
Is this refactor really necessary? only saves 2 lines of code.

Reply:
This is for DOM. If `timestamp` is falsy, we don't need to add `<span>` to DOM, which is very costly.

Reply:
Better have a single `return`.